shared/js/background/tab-manager.es6.js

```diff
@@ -44,6 +44,7 @@ class TabManager {
     }
 
     delete (id) {
+        this.tabContainer[id]?.adClick?.adClickDNR?.removeAdClickDNR()
```

Comment:
Same here I think a remove method on adClick would make the most sense rather than many consumers know the internals of the class structures.